pdf/pdfcanvas.cpp

```diff
@@ -278,9 +278,16 @@ void PDFCanvas::layout()
     update();
 }
 
-static qreal squaredDistanceFromRect(const QRectF &rect, const QPoint &point)
+qreal PDFCanvas::squaredDistanceFromRect(const QRectF &pageRect, const QRectF &redRect,
```

Comment:
Ah sorry always use abreviations that we use at work without thinking about it. Red is for reduced (*i.e.* coordinates in [0;1]). Could change redRect to reducedCoordRect. Still abbreviated but maybe clearer ?